src/plots/plots.js

```diff
@@ -2617,7 +2617,7 @@ plots.doCalcdata = function(gd, traces) {
     }
 
     // clear stuff that should recomputed in 'regular' loop
-    if(hasCalcTransform) clearAxesCalc(axList);
+    if(hasCalcTransform) setupAxisCategories(axList);
```

Comment:
`, fullData`?

Reply:
maybe a can of worms, but is there a test for multicat + calctransforms (or non-calc-transforms for that matter)?

Reply:
Multicategory axes don't support transforms. I'm not sure how gracefully things fail though.

Reply:
OK, that's fine. Nothing more from me - I'll let one of the other folks give this a once-over and the 💃

Reply:
Transforms + multicategory tracked in -> https://github.com/plotly/plotly.js/issues/3363